crates/server-config/src/node_config.rs

```diff
@@ -85,15 +85,20 @@ pub struct NodeConfig {
     #[serde(default = "default_aquavm_pool_size")]
     pub aquavm_pool_size: usize,
 
-    /// Maximum heap size in bytes available for the module.
+    /// Maximum heap size in bytes available for the VM.
     #[serde_as(as = "Option<DisplayFromStr>")]
     #[serde(default)]
     pub aquavm_max_heap_size: Option<bytesize::ByteSize>,
 
+    /// Maximum heap size in bytes available for the module.
+    #[serde_as(as = "Option<DisplayFromStr>")]
+    #[serde(default)]
+    pub app_service_max_heap_size: Option<bytesize::ByteSize>,
```

Comment:
Since not a service-wise setting, it's applied to each module individually. Maybe rename like this?
```suggestion
    pub service_module_max_heap: Option<bytesize::ByteSize>,
```

or `wasm_module_max_heap` or `module_max_heap`